tests/test_image/test_photometric.py

```diff
@@ -200,3 +200,21 @@ def _adjust_contrast(img, factor):
                 _adjust_contrast(img, factor).astype(np.int32),
                 rtol=0,
                 atol=1)
+
+    def test_lut_transform(self):
+        lut_table = np.array(list(range(256)))
+
+        img = mmcv.lut_transform(self.img, lut_table)
+        assert id(img) != id(self.img)
```

Comment:
Why testing `id`?

Reply:
Test whether the original image is changed.